src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -397,6 +350,23 @@ typedef struct {
     char kvsname[MPIDI_KVSAPPSTRLEN];
     char pname[MPI_MAX_PROCESSOR_NAME];
     int port_name_tag_mask[MPIR_MAX_CONTEXT_MASK];
+
+    /* Capability settings */
+#ifdef MPIDI_OFI_ENABLE_RUNTIME_CHECKS
```

Comment:
I don't necessarily agree with changing `runtime` to `any`. Just because we say `runtime` doesn't mean we want *any* provider. We might pass in `runtime` and set the CVAR. To me, that means I want to pick the provider at runtime, not pick any provider available. This is a minor thing though and my rationale for picking names usually sucks. So if you want `any`, that's fine.